### DISCUSSION TITLE
PWA IOS startup image

### DISCUSSION THREAD

**Discussion by Egorles** [arcomage/arcomage-hd#89] [General]
Great project, you did big work! 
I found the repo looking for PWA with working startup image for IOS, and it works in your project! My respects, cause in very few apps this works for IOS. So I copied manifest.json and all icons but it still doest work for me! Have you had any problems with it or it just worked? May be you have any tips? 
Thank you!

**Comment by tomchen** [comment]
Thanks.

I don't recall too many difficulties, but manifest.json shouldn't be the only thing you need to do. You can search "pwacompat" and "workbox" in my code to see what else to do.
